zebra/rt_netlink.c

```diff
@@ -142,7 +142,7 @@ static uint16_t neigh_state_to_netlink(uint16_t dplane_state)
 }
 
 
-static inline int is_selfroute(int proto)
+static inline bool is_selfroute(int proto)
```

Comment:
make it `return true` and `return false` as well.  Sorry I did not spot this the first time.

Reply:
My bad, I did not look into it well as well. Working on it now.